servers/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -47,10 +56,18 @@ public static void main(String[] args) {
 
     try {
       worker.process();
+      // Start the lineage worker
+      if (LineageUtils.isLineageEnabled(WorkerContext.getConf())) {
+        lineageWorker.start();
```

Comment:
I don't think lineage worker can start, because worker.process will start Thrift server (by `mThriftServer.serve()`), which blocks all the subsequent calls. So you need to start Lineage worker before that. Also can we move web server and thrift server into server base to share with all types of worker?